### ISSUE TITLE
Collect metrics for WiFi

### ISSUE THREAD

**Issue by vpetersson** [databat-io/sonar#10]
BLE provides good insight, but we can also collect metrics from WiFi devices in the surrounding area.

**Comment by cyberman54** [comment]
see here:
https://github.com/cyberman54/ESP32-Paxcounter

**Comment by vpetersson** [comment]
Interesting, but I don’t want to add any additional hardware. It should be possible (in theory) to get this data using the built-in WiFi chipset in the Raspberry Pi 3 Model B (and later).

**Comment by cyberman54** [comment]
No, it's not, since the builtin Wifi chip in RPi3 does not support monitor mode. But it would be possible to add a suitable wifi chip as usb dongle.

**Comment by vpetersson** [comment]
Interesting. Will take a look at this later on.

**Comment by Sean-Law** [comment]
@vpetersson is this project still under development?

**Comment by vpetersson** [comment]
@Sean-Law Not actively being worked on (as i don't have the bandwidth) but should still work.
